docs/source/checks/nlp/data_integrity/plot_under_annotated_property_segments.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+"""
+
+.. _nlp__under_annotated_property_segments:
+
+Under Annotated Property Segments
+*********************************
+
+This notebook provides an overview for using and understanding the under annotated property segments check.
+
+**Structure:**
+
+* `What is the purpose of the check? <#what-is-the-purpose-of-the-check>`__
+* `Automatically detecting under annotated segments <#automatically-detecting-under-annotated-segments>`__
+* `Generate data & model <#generate-data-model>`__
+* `Run the check <#run-the-check>`__
+* `Define a condition <#define-a-condition>`__
+
+What is the purpose of the check?
+==================================
+
+The Under-Annotated Property Segments check is designed to help you easily identify segments in your data which are
+under-annotated compared to the rest of your dataset, based on the provided :ref:`properties <nlp__properties_guide>`.
+The check could be very useful for example for identifying a specific data samples for which there was
```

Comment:
Not really an example, maybe (the example I give in the webinar) - less fluent or less formal samples are harder to label, so the annotators are avoiding labeling them?